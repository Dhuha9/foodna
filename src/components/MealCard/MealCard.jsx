/* eslint react/prop-types: 0 */
// eslint-disable-next-line react/jsx-no-bind
import React, { Component } from 'react';
import '../../meals.css'
import { Button } from 'react-bootstrap';


export default class MealCard extends Component {
  render() {
    return (
      <div className="card">
        <div className="image-section">
          <img src={this.props.img} className="card-img" alt="food" />
        </div>
        <div className="card-body">
          <h5 className="card-title">{this.props.title}</h5>
          <span className="card-text">{this.props.organization}</span>
          <Button className="Button">Get This Meal</Button>
        </div>
      </div>
    );
  }
}
