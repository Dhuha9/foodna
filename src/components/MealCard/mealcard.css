.card{
    height: 436px;
    width: 270px;
    top: 247px;
    left: 135px;
}

.card-body{
    position: relative;
}

.Button{
    background-color: #E5E5E5;
    height: 60px;
    width: 100%;
    border: 0;
    border: none;
    color: black;
    position: absolute;
    bottom: 0;
    left: 0;
}

.Button:hover {
    color: #fff;
    background-color: #03B664;
    border-color: #0062cc;
}

.card-img {
    width: 270px;
    height: 270px;
    object-fit: cover;
}

.card-title{
    position: absolute;

}

.card-text{
    position: absolute;
    bottom: 80px;
    left: 18px;
    color: gray;
}

