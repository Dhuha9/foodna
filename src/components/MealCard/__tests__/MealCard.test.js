import * as React from 'react';
import * as renderer from 'react-test-renderer';
import MealCard from '../MealCard';
it('renders apply now correctly', () => {
  const tree = renderer.create(<MealCard />).toJSON();
  expect(tree).toMatchSnapshot();
});