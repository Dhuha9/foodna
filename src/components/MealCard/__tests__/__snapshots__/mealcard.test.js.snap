// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders apply now correctly 1`] = `
<div
  className="card"
>
  <div
    className="image-section"
  >
    <img
      alt="food"
      className="card-img"
    />
  </div>
  <div
    className="card-body"
  >
    <h5
      className="card-title"
    />
    <span
      className="card-text"
    />
    <button
      className="button"
      type="button"
    >
      Get This Meal
    </button>
  </div>
</div>
`;
